### PR TITLE
docs: Update identity-map.md

### PR DESCRIPTION
**Pull request by phuffer**
Fix typo and ambiguous use of `it`.

### CODE REVIEW

File: docs/versioned_docs/version-6.1/identity-map.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ app.use((req, res, next) => {
 
 Middlewares are executed only for regular HTTP request handlers, what if you need a request scoped method outside that? One example of that is queue handlers or scheduled tasks (e.g. CRON jobs).
 
-In those cases, you can use the `@CreateRequestContext()` decorator. It requires you to first inject the `MikroORM` instance (or an `EntityManager` or some `EntityRepository`) to current context, it will be then used to create a new context for us. Under the hood, the decorator will register the new request context for our method and execute it inside it (vi `RequestContext.create()`).
+In those cases, you can use the `@CreateRequestContext()` decorator. It requires you to first inject the `MikroORM` instance (or an `EntityManager` or some `EntityRepository`) to current context, it will be then used to create a new context for us. Under the hood, the decorator will register the new request context for our method and execute it inside the method (via `RequestContext.create()`).
```
[the remark this answers is not in the document]
Comment (B4nan) [reply docs/versioned_docs/version-6.1/identity-map.md:114]: The `it` refers to the context - the method is executed inside the context.

[the remark this answers is not in the document]
Comment (phuffer) [reply docs/versioned_docs/version-6.1/identity-map.md:114]: Ah okay, should it read `... and execute the method inside it (via `RequestContext.create()`).`? Or maybe something like `... register the new request context for our method and execute it inside the context (via `RequestContext.create()`).` Or I can leave it as is.

[the remark this answers is not in the document]
Comment (B4nan) [reply docs/versioned_docs/version-6.1/identity-map.md:114]: Yeah, both sound good to me.

[the remark this answers is not in the document]
Comment (phuffer) [reply docs/versioned_docs/version-6.1/identity-map.md]: Updated. Thank you!

**Review by B4nan — APPROVED** [review]

**Pull request merged**